desktop/apps/consignments2/client/reducers.js

```diff
@@ -1,8 +1,10 @@
 import * as actions from './actions'
 import u from 'updeep'
+import { clone, merge } from 'lodash'
```

Comment:
Looking at `package.json`, I don't think Force uses `lodash` (with the exception of `lodash.get`, which is plucked from the main package with no dependencies), only `underscore`.